### DISCUSSION TITLE
欢迎来到pbrt中译版讨论区

### DISCUSSION THREAD

**Discussion by kanition** [kanition/pbrtbook#3] [公告]
# 👋 欢迎来到pbrt中译版讨论区！
##  学习有困难？没关系！你可以在此处针对本书进行：
  * 提出和本书内容相关的问题；
  * 解答他人的问题；
  * 报告本书翻译和内容错误；
  * 提出有益的建议或意见；
  * 展示你的自学成果，分享学习心得与笔记；
  * 发表其他你认为相关的话题。
  
### 第三版
* 原书官网：[http://www.pbr-book.org/](http://www.pbr-book.org/)
* 原书源码：[https://github.com/mmp/pbrt-v3](https://github.com/mmp/pbrt-v3)
### 第四版
* 原书源码：[https://github.com/mmp/pbrt-v4](https://github.com/mmp/pbrt-v4)

**Comment by kanition** [comment]
【友情链接】GAMES101: 现代计算机图形学入门：https://sites.cs.ucsb.edu/~lingqi/teaching/games101.html

【友情链接】一周入门光线追踪<a href="https://raytracing.github.io/books/RayTracingInOneWeekend.html">
    <cite>Ray Tracing in One Weekend</cite>
</a>

**Comment by zhangf911** [comment]
这个翻译真不错，还补充了不少参考资料。期待更新。另外除了这边还有其他沟通交流的地方吗？不仅仅是书本身，比如实践中遇到的一些问题交流

**Reply by kanition** [reply]
感谢对本书的喜爱！
讨论区就是关于本书译文的主要交流地点，也方便其他有类似疑问的伙伴检索查阅。笔者觉得这比QQ群、微信群等形式更为合适。
原谅笔者也是初学者，实践方面没有经验，可能无法给出什么好的建议。如果想要交流实践问题，建议前往第四版pbrt代码仓库提问。该区活跃度比本书讨论区高得多，而且原作者会经常亲自回答。
地址：https://github.com/mmp/pbrt-v4/issues

**Comment by kanition** [comment]
由@mumu-sild 建立的学习交流QQ群号：734217395，欢迎大家加入
